### PR TITLE
creating graphql schema

### PR DESCRIPTION
**Pull request by gsasouza**
Criação do schema.graphql e schema.json para usar no client

### CODE REVIEW

File: data/schema.graphql [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Person data
+type Person {
+  # The ID of an object
+  id: ID!
+  _id: Int
```
Comment (jgcmarins) [inline data/schema.graphql]: Hmmm, why int?

**Review by jgcmarins — APPROVED** [review]
Looks good. Keep going :)

File: data/schema.graphql (hunk 1 above)
Comment (gsasouza) [reply data/schema.graphql]: My bad, this have to be a String (cause mongo's ObjectId can be represented as a String). But, i'm consuming a public api and I don't even use mongo. Nice alert, i'll remove this field

Comment (jgcmarins) [reply data/schema.graphql]: no problem :)
Do you know that `schema.graphql` is a generated file, right?

Comment (jgcmarins) [reply data/schema.graphql]: anyway, the fix must come from your schema definition type: https://github.com/gsasouza/starwars-api-graphql-wrapper/blob/master/src/PeopleType.js#L22

Comment (gsasouza) [reply data/schema.graphql]: Yes, i used the updateSchema.js (copy-pasted from your repo) to generate schema.graphql and schema.json files. I'm following this tutorial https://facebook.github.io/relay/docs/tutorial.html

Comment (jgcmarins) [reply data/schema.graphql]: nice job
keep going

File: src/schema.js [hunk 2]
```diff
@@ -78,10 +74,14 @@ const QueryType = new GraphQLObjectType({
...
       type: new GraphQLList(PersonType),
-      resolve: async () => {
-        const resp = await fetch('https://swapi.co/api/people/');
+      args: {
+        page: {
+          type: GraphQLInt
+        }
+      },
+      resolve: async (_, args) => {
+        const resp = await fetch(`https://swapi.co/api/people/?page=${args.page}`);
```
Comment (jgcmarins) [inline src/schema.js]: nice

File: src/app.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import "babel-polyfill"
 import Koa from 'koa'
 import graphqlHttp from 'koa-graphql'
 
-import { schema } from './PeopleType'
+import { schema } from './schema'
```
Comment (jgcmarins) [inline src/app.js:5]: It's not that good, but it's ok.
The best thing to do is to keep `PeopleType`

**Review by jgcmarins — APPROVED** [review]
good job

**Review by jgcmarins — APPROVED** [review]
good

**Pull request merged**